ajax/libs/covervid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "covervid",
+  "filename": "covervid.min.js",
+  "version": "1.0.0",
+  "description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin extension",
+  "author": "Stefan Erickson",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stefanerickson/covervid"
+  },
+  "keywords": [
```

Comment:
Could you modify the keywords? It will be better to let it as the same with the upstream.
ref: https://github.com/stefanerickson/covervid/blob/gh-pages/bower.json#L10-L15